README - INSTALL INSTRUCTIONS/QBOX/README.md

```diff
@@ -70,22 +80,26 @@ local function spawnLastLocation()
 
     TriggerServerEvent('QBCore:Server:OnPlayerLoaded')
     TriggerEvent('QBCore:Client:OnPlayerLoaded')
-    -- TriggerServerEvent('qb-houses:server:SetInsideMeta', 0, false)
-    -- TriggerServerEvent('qb-apartments:server:SetInsideMeta', 0, 0, false)
 
     while not IsScreenFadedIn() do
         Wait(0)
     end
 end
 ```
-Edit in config/server.lua in `characterDataTables`:
 
-`{'properties', 'owner'}` to `{'properties', 'owner_citizenid'}`
+## `config/server.lua`
+
+### Step 1: Update `characterDataTables`
 
+Change `{'properties', 'owner'}` in `characterDataTables = {` to `{'properties', 'owner_citizenid'}`.
 
-# qbx_spawn Resource Edits
+# QBX Spawn
 
-Find `inputHandler` in client/main.lua and replace with: 
+## `client/main.lua`
```

Comment:
This can be deleted when describing the file and putting the file name in the code (see other review comments)